beamer/metrics.py

```diff
@@ -19,8 +19,8 @@ def init(config: Any) -> None:
     info.info(
         dict(
             account=config.account.address,
-            l2a_rpc_url=config.l2a_rpc_url,
-            l2b_rpc_url=config.l2b_rpc_url,
+            l2a_rpc_url=source_rpc_url,
+            l2b_rpc_url=target_rpc_url,
```

Comment:
We should also rename `l2a_rpc_url` and `l2b_rpc_url` to `source_rpc_url` and `target_rpc_url`, respectively.

Reply:
this may create some problems in prometheus if we are using it with some calculations etc.

Reply:
> this may create some problems in prometheus if we are using it with some calculations etc.

Our Prometheus instance is no longer tracking beamer-agent so nothing will break.
I also seriously doubt that any of our agents use this.

Reply:
i guess it is not a problem as this is not posting data to a central prometheus or so